src/pages/Rocket/Rocket.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import { bookRockets, cancelBooking } from '../../redux/Rockets/Rockets';
+
+const Rocket = (props) => {
+  const rocket = props;
+  const dispatch = useDispatch();
+
+  const bookRocket = (id) => {
+    console.log(id);
```

Comment:
- [ ] [OPTIONAL] I suggest you remove console statements in your code. The purpose of console statements is debugging. After you are done debugging your code, it's best practice to remove any dead code from production code. Putting a console statement in your code may reduce the performance of the program